addons/mail/static/src/composer/composer.xml

```diff
@@ -11,7 +11,7 @@
                     'px-3 py-4 o-extended': extended,
                     'p-3': normal,
                     'o-hasSelfAvatar px-3': !env.inChatWindow and thread,
-                }">
+                }" t-attf-class="{{ props.className }}">
```

Comment:
mixing single line and multiple lines, maybe a new line?

Reply:
IMO I'm not fond of spacing too much, because I have to scroll.
This is also a fix, so better make minimal diff, so that there are less conflicts in follow-up PR